handel/src/services/kms/index.ts

```diff
@@ -90,7 +90,7 @@ function getDefaultAlias(serviceContext: ServiceContext<KmsServiceConfig>): stri
  */
 
 export function check(serviceContext: ServiceContext<KmsServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
```

Comment:
You're missing the import for the `checkPhase` module, so the compile step of the build is failing. Make sure to run `npm run build` when you make changes prior to pushing up the branch; that will help you catch these kinds of compile errors.